packages/website/utils/getGridStyles.ts

```diff
@@ -0,0 +1,28 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const TOPBAR_HEIGHT = '70px';
+export const BIG_SCREEN_BREAKPOINT = '1600px';
```

Comment:
just a detail, dunno why, but for some reason, `BIG_SCREEN_BREAKPOINT` sounds a bit strange to me 🙂
Maybe we could do rename it to something like `SCREEN_BREAKPOINT_XL` or `SCREEN_BREAKPOINT_LARGE`.